nrf_security/doc/driver_config.rst

```diff
@@ -0,0 +1,344 @@
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+| CTR            | :kconfig:`PSA_CRYPTO_DRIVER_ALG_CTR_CC3XX`            | :kconfig:`PSA_CRYPTO_DRIVER_ALG_CTR_OBERON`            |
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+| OFB            | Not supported                                         | Not supported                                          |
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+| XTS            | Not supported                                         | Not supported                                          |
++----------------+-------------------------------------------------------+--------------------------------------------------------+
+
+.. note::
+   * If an AES cipher mode is enabled and no PSA driver enables or supports it, then builtin Mbed TLS support is enabled and used.
+   * The :ref:`nrf_security_drivers_cc3xx` is limited to AES key sizes of 128 bits on devices with Arm CryptoCell cc310.
+
+
+MAC configuration
+*****************
+
+MAC support can be enabled by setting one or more Kconfig variables in the following table:
+
++----------------+------------------------------+
+| MAC cipher     | Configurations               |
++================+==============================+
+| ECB_NO_PADDING | :kconfig:`PSA_WANT_ALG_CMAC` |
++----------------+-------------------------------
+| CBC_NO_PADDING | :kconfig:`PSA_WANT_ALG_HMAC` |
++----------------+-------------------------------
+
+
+MAC driver configurations
+=========================
+
+The Kconfig variables in the following table can be used for fine-grained control over which drivers provides AEAD support:
+
++----------------+---------------------------------------------+----------------------------+
+| MAC cipher     | nrf_cc3xx driver support                    | nrf_oberon driver support  |
++================+=============================================+============================+
+| ECB_NO_PADDING | :kconfig:`PSA_CRYPTO_DRIVER_ALG_CMAC_CC3XX` | Not supported              |
++----------------+----------------+----------------------------+----------------------------+
+| CBC_NO_PADDING | :kconfig:`PSA_CRYPTO_DRIVER_ALG_HMAC_CC3XX` | Not supported              |
++----------------+----------------+----------------------------+----------------------------+
+
+.. note::
+   * If a MAC algorithm is enabled and no PSA driver enables or supports it, then builtin Mbed TLS support is enabled and used.
+   * The :ref:`nrf_security_drivers_cc3xx` is limited to AES CMAC key sizes of 128 bits on devices with Arm CryptoCell cc310.
+   * The :ref:`nrf_security_drivers_cc3xx` is limited to HMAC using SHA-1, SHA-224, and SHA-256 on devices with Arm CryptoCell cc310.
```

Comment:
which hashing algorithms are available outside of cc310 ? We should mention both what is supported and what is not supported.